data/styles/vehicle/include/priorities_4_overlays.prio.txt

```diff
@@ -350,71 +350,71 @@ railway-station-subway-hamburg                      # icon z13- (also has captio
 railway-station-subway-pyongyang                    # icon z13- (also has caption(optional) z13-)
 railway-station-subway-rennes                       # icon z13- (also has caption(optional) z13-)
 railway-station-subway-rio                          # icon z13- (also has caption(optional) z13-)
-railway-station-subway-roma                         # icon z13- (also has caption(optional) z13-)
+railway-station-subway-roma                         # icon z12- (also has caption(optional) z13-)
 railway-station-subway-rotterdam                    # icon z13- (also has caption(optional) z13-)
 railway-station-subway-samara                       # icon z13- (also has caption(optional) z13-)
 railway-station-subway-santiago                     # icon z13- (also has caption(optional) z13-)
 railway-station-subway-santo_domingo                # icon z13- (also has caption(optional) z13-)
 railway-station-subway-saopaulo                     # icon z13- (also has caption(optional) z13-)
 railway-station-subway-sapporo                      # icon z13- (also has caption(optional) z13-)
 railway-station-subway-sendai                       # icon z13- (also has caption(optional) z13-)
-railway-station-subway-sf                           # icon z13- (also has caption(optional) z13-)
-railway-station-subway-shanghai                     # icon z13- (also has caption(optional) z13-)
+railway-station-subway-sf                           # icon z12- (also has caption(optional) z13-)
+railway-station-subway-shanghai                     # icon z12- (also has caption(optional) z13-)
 railway-station-subway-shiraz                       # icon z13- (also has caption(optional) z13-)
 railway-station-subway-singapore                    # icon z13- (also has caption(optional) z13-)
 railway-station-subway-sofia                        # icon z13- (also has caption(optional) z13-)
-railway-station-subway-stockholm                    # icon z13- (also has caption(optional) z13-)
+railway-station-subway-stockholm                    # icon z12- (also has caption(optional) z13-)
 railway-station-subway-tabriz                       # icon z13- (also has caption(optional) z13-)
 railway-station-subway-taipei                       # icon z13- (also has caption(optional) z13-)
 railway-station-subway-taoyuan                      # icon z13- (also has caption(optional) z13-)
 railway-station-subway-tashkent                     # icon z13- (also has caption(optional) z13-)
 railway-station-subway-tbilisi                      # icon z13- (also has caption(optional) z13-)
 railway-station-subway-tehran                       # icon z13- (also has caption(optional) z13-)
 railway-station-subway-tianjin                      # icon z13- (also has caption(optional) z13-)
-railway-station-subway-tokyo                        # icon z14- (also has caption(optional) z14-)
+railway-station-subway-tokyo                        # icon z14- and caption z13 (also has caption(optional) z14-)
```

Comment:
some weird changes here (a caption w/o an icon added to z13), in the main style its 
`# icon z14- (also has caption(optional) z14-)`

I bet its because in the vehicle and main styles the "default" subway style definition is different in Icons.mapcss, prob need to be synced